src/licensedcode/data/licenses/bsd-advertising-acknowledgement.LICENSE

```diff
@@ -0,0 +1,56 @@
+---
+key: bsd-advertising-acknowledgement
+short_name: BSD Advertising Acknowledgement License
+name: BSD Advertising Acknowledgement License
+category: Permissive
+owner: David Giffin
+spdx_license_key: BSD-Advertising-Acknowledgement
+other_urls:
+    - https://github.com/python-excel/xlrd/blob/master/LICENSE#L33
+ignorable_copyrights:
+    - Copyright (c) 2001 David Giffin
+ignorable_holders:
+    - David Giffin
+ignorable_authors:
+    - David Giffin <david@giffin.org>
+ignorable_emails:
+    - david@giffin.org
+---
+
+Copyright (c) 2001 David Giffin.
+All rights reserved.
```

Comment:
Please, remove this copyrights from the reference license text (lines 20 and 21)

Also, https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/other-permissive_300.RULE needs to be updated to report bsd-advertising-acknowledgement

Reply:
Done!